cdap-ui/app/cdap/components/Header/Tour/index.ts

```diff
@@ -23,28 +27,56 @@ const steps: ITourStep[] = [
     id: 'welcome',
     title: 'Welcome to CDAP',
     text: [
-      'CDAP is an open source framework that simplifies data application development,' +
-      ' data integration and data management.',
+      'CDAP is an open source framework that simplifies data application development, data integration,' +
```

Comment:
Any specific reason why this alone is not in i18n?

Reply:
Yeah, this first step will eventually be removed. It will become a welcome modal